internal/resources/templates/bids_results.go

```diff
@@ -64,6 +66,20 @@ const BidsResults = `
 	<div>Size: {{ .Summary.Size }}</div>
 {{ end }}
 		</div>
+		<div class="column" style="width:35%">
+			<div id="history">History:</div>
+			{{ range $val := .Results }}
+			<div>
+				<a href="{{$val.Href}}" alt="{{$val.Alt}}">
+					{{$val.Badge}}
+					<span class="date-tiny">{{$val.Text1}}</span>
```

Comment:
Please merge both spans into one with padding: `<span style="padding-left:20px">{{$val.Text1}} {{$val.Text2}}</span>` or add the padding to the css.